CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" ON)
 
 option(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R "Enable experimental YCgCo-R matrix code" OFF)
+option(AVIF_ENABLE_EXPERIMENTAL_AVIR "Enable experimental reduced header" OFF)
```

Comment:
Nit: Please add a comment somewhere (either in this file or in avif.h) to specify the ISO proposal/contribution that specifies the reduced HEIF header.

Reply:
Note: I added m64572 for HEIF only as there is no specification amendment proposal on the AVIF side for now.